index.bs

```diff
@@ -549,15 +549,13 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 : <dfn>Authenticator</dfn>
 : <dfn>[WAA]</dfn>
-:: A cryptographic entity used by a [=[WAC]=] to (i) generate a [=public key credential=] and register it with a [=[RP]=],
-    and (ii) [=authentication|authenticate=] by potentially [=user verification|verifying the user=], and then 
-    cryptographically signing and returning, in the form of an [=Authentication Assertion=], 
-    a challenge and other data presented by a [=[WRP]=] (in concert with the [=[WAC]=]).
+:: A cryptographic entity, existing in hardware or software, that can [=registration|register=] a user with a given [=[RP]=] 
+    and later [=Authentication Assertion|assert ownership=] of the registered [=public key credential=] when requested by the [=[RP]=]. 
+    [=Authenticators=] can report information regarding their [=authenticator types|type=] and security characteristics via [=attestation=] 
+    during [=registration=].
 
-    A [=[WAA]=] could be, for example, a dedicated USB or wireless device separate from the [=client device=],
-    a dedicated hardware subsystem integrated into the [=client device=],
-    or a software component of the [=client=].
-    [=Authenticators=] can report their type and security characteristics via [=attestation=].
+    A [=[WAA]=] could be, for example, a [=roaming authenticator|USB or wireless device=] that is attached to the [=client device=],
```

Comment:
Roaming authenticators are not attached. They are separate and communicate via various transports and need not be repeated here. Directly link to the definition.  

Change it to:
  A [=[WAA]=] could be, for example, a [=roaming authenticator=], a dedicated hardware subsystem in …